checker-qual/src/main/java/org/checkerframework/checker/mustcallonelements/qual/MustCallOnElementsUnknown.java

```diff
@@ -0,0 +1,23 @@
+package org.checkerframework.checker.mustcallonelements.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.SubtypeOf;
+
+/**
+ * The top qualifier in the Must Call type hierarchy. It represents a type that might have an
+ * obligation to call any set (even an infinite set!) of methods. This type contains every object.
+ * This type should rarely be written by a programmer.
+ *
+ * <p>The Object Construction Checker cannot verify that the property represented by this annotation
```

Comment:
https://github.com/typetools/checker-framework/pull/6687 also made a change to the text from which this was copied to remove the reference to the OCC, and a corresponding change should be made here.